enhancements/sdk-fbc-run-bundle.md

```diff
@@ -0,0 +1,279 @@
+
+10. In this way, the File-Based Catalog that was generated by the Operator SDK will be served to the gRPC API via the specified port (defaulted to `50051`) without any user-facing changes.
+
+11. The remaining resources (`CatalogSource`, `Subscription`, `OperatorGroup`) are created in the same way as before. The run bundle command does not change from here on.
+
+- Note: If a SQLite index is passed, the old commands will be used in the registry pod and none of the FBC creation will happen. Below is the container creation command used to handle SQLite index images (the optional flags are skipped here for simplicity purposes):
+
+    ```go
+    `mkdir -p {{ dirname .DBPath }} && \
+    {{- range $i, $item := .BundleItems }}
+    opm registry add -d {{ $.DBPath }} -b {{ $item.ImageTag }} --mode={{ $item.AddMode }} && \
+    {{- end }}
+    opm registry serve -d {{ .DBPath }} -p {{ .GRPCPort }}`
+    ```
+
+#### Example UX Scenarios
+
+Running the following scenarios results in different outcomes for FBC creation. These are some example uses of the `run bundle` command. This is a not an exhaustive list of the command with the other flags.
+
+```sh
+- operator-sdk run bundle <bundle-image>
+- operator-sdk run bundle <bundle-image> --index-image quay.io/example/testFBCCatalog
+- operator-sdk run bundle <bundle-image> --index-image quay.io/example/testSQLiteCatalog
+```
+
+- `operator-sdk run bundle <bundle-image>` would create a File-Based Catalog (FBC) on the fly by rendering the bundle image and using the default upstream opm image `quay.io/operator-framework/opm:latest`. Bundle image contents will be served over a gRPC port through the ephemeral registry pod.
+- `operator-sdk run bundle <bundle-image> --index-image quay.io/example/testFBCCatalog` would ensure that the bundle image doesn't exist in the `testFBCCatalog` index. If it exists, then we error out. If not, we add the bundle image to the index image by rendering both of these images using the Operator Registry APIs thereby appending the bundle contents to the index and serving the index over the gRPC port.
+- `operator-sdk run bundle <bundle-image> --index-image quay.io/example/testSQLiteCatalog` will still be supported in the old way by making use of `opm registry add` and `opm registry serve` commands under the hood and without creating an FBC or rendering the SQLite based image.
+
+#### Prototype Links
+
+- [PR](<https://github.com/operator-framework/operator-sdk/pull/5671>) that implements FBC support in run bundle.
+- [run bundle prototype branch](<https://github.com/rashmigottipati/operator-sdk/tree/run-bundle-prototype>).
+- Initial [POC](<https://github.com/rashmigottipati/run-bundle-POC>) to create a declarative config using Operator Registry APIs.
```

Comment:
what is the diff between both prototypes? why do we have 2 prototypes? 
where/how each one of them is relevant to who will read this EP? I mean, what should we check in each case?

Reply:
PR and the run bundle prototype branch both refer to the working branch where we have got `run bundle` to work with File-Based Catalog. The initial POC is just something we tried out initially to create a declarative config using Operator Registry APIs (which eventually we used for creating the PR)

Reply:
IMO that would be under Implementation History and they we would say something like:

Check [here](link) a POC to verify how to create a declarative config using Operator Registry APIs. Also, check [here] ... and by last the actually open PR with its changes which is a WIP.

Reply:
I moved the actual PR link to implementation history and left the prototypes here.